src/pybind/mgr/cephadm/module.py

```diff
@@ -1402,18 +1414,70 @@ def get_osd_uuid_map(self, only_up=False):
                 r[str(osd_id)] = o.get('uuid', '')
         return r
 
+    def resolve_hosts_for_osdspecs(self,
+                                   specs: Optional[List[DriveGroupSpec]] = None,
+                                   service_name: Optional[str] = None
+                                   ) -> List[str]:
+        osdspecs = []
+        if service_name:
+            self.log.debug(f"Looking for OSDSpec with service_name: {service_name}")
+            osdspecs = self.spec_store.find(service_name=service_name)
+            osdspecs = [cast(DriveGroupSpec, spec) for spec in osdspecs]
```

Comment:
this line doesn't do anything. Which mypy error do you solve here?

Reply:
right, this can go away